assets/src/edit-story/components/form/dateTime/timeZone.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { getSettings, format } from '@web-stories-wp/date';
+import getTimeZoneString from '@web-stories-wp/date/src/getTimeZoneString';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import Tooltip from '../../tooltip';
+import { PLACEMENT, Text, THEME_CONSTANTS } from '../../../../design-system';
+
+const Wrapper = styled.div``;
+
+const StyledText = styled(Text)`
+  color: ${({ theme }) => theme.colors.fg.tertiary};
+  line-height: 30px;
+`;
+
+function TimeZone({ date }) {
+  const { timezone } = getSettings();
+
+  const zoneAbbr = timezone?.length
+    ? format(date, 'T')
+    : `UTC${getTimeZoneString()}`;
```

Comment:
@swissspidy Should this be translatable, too, 'UTC'?

Reply:
I don't think so. Gutenberg [doesn't do that either](https://github.com/WordPress/gutenberg/blob/fa457877f45abb692369addab76d579d5d31539d/packages/components/src/date-time/timezone.js#L18-L36).

On that note, Gutenberg's date picker has some nice logic in it (see link above):

1. Don't display timezone information if user is in the same timezone
2. Use "Coordinated Universal Time" in the tooltip if the timezone is set to UTC

Those seem like two useful enhancements to me, that I think we can apply here as well.